.github/workflows/api-docker-publish-releases.yml

```
@@ -17,7 +17,8 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Build api image
-        run: docker build ./packages/api --file Dockerfile --tag image \
+        working-directory: "./packages/api"
+        run: docker build --file Dockerfile --tag image
```

Siden våre docker filer heter Dockerfile så file flagget fjernes. 

```suggestion
        run: docker build . --tag image
```